Makefile

```diff
@@ -0,0 +1,42 @@
+PROG_NAME=olm-mermaid-graph
+#OUTPUT_TYPE=png # or pdf
+OUTPUT_TYPE=svg
+MERMAID_TEMP_SCRIPT=mermaid.mer
+#INDEX_DB_PATH_AND_NAME?=/Users/btofel/workspace/sample-operator/test-registry.db
+#INDEX_DB_PATH_AND_NAME?=olm_catalog_indexes/index.db.4.6.community-operators
+INDEX_DB_PATH_AND_NAME?=olm_catalog_indexes/index.db.4.6.redhat-operators
```

Comment:
Could we add comments to explain when it option is useful. I mean, did you keep that because they are useful for someone tests I am right? Another option would have makefile targets that use them with their specific comment to explain when/why to use them.

Reply:
Before that please see: https://github.com/operator-framework/index-mermaid-graph/pull/1/files#r612848316

Reply:
Added comments in bd65b403

And on shipping the index database files with the repo: It's a little bulky the way it's being done now, but I think as a first-pass this assures users all get some version of the indexes to play around with and everyone knows what the head-end of all this is currently -- the index database files. I agree your solution for the audit is a better way to go long term.